runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -286,7 +252,6 @@ public void onMenuOptionClicked(MenuOptionClicked event)
 			|| event.getId() == ItemID.SUPERANTIPOISON4))
 		{
 			createGameTimer(SUPERANTIPOISON);
-			return;
```

Comment:
:-1: If ever more code is added below this check, it'd be easier to leave this statement in than to expect the new code re-add it.